src/client/App.js

```diff
@@ -1,36 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import './assets/fonts/fonts.css';
-
 import { Home } from './containers/Home/Home.container';
 import SignIn from './containers/SignIn';
 import SignUp from './containers/SignUp';
 import ResetPassword from './containers/ResetPassword';
 import AuthenticatedRoute from './components/Auth/AuthenticatedRoute.component';
 import { useAuthentication } from './hooks/useAuthentication';
-
 import Profile from './containers/Profile';
 import Loader from './components/Loader';
-
+import { QuizResultPage } from './containers/QuizResultPage/QuizResultPage.component';
+import CareerPage from './containers/CareerPage/CareerPage.component';
 function App() {
   const { isLoading } = useAuthentication();
-
+  const [career, setCareer] = useState('');
+  const updateCareer = (title) => {
+    setCareer(title);
+  };
```

Comment:
Why is the here? and not in the `<QuizResultPage>` component?

Reply:
we have to discuss it with you

Reply:
we have fixed it